src/prometheus/steps/AntiTamper.lua

```diff
@@ -105,7 +105,7 @@ function AntiTamper:apply(ast, pipeline)
     end
     code = code .. [[
     local gmatch = string.gmatch;
-    local err = function() error("Tamper Detected!") end;
+    local err = function() return print("Tamper Detected!") end;
```

Comment:
This is good but a better way would be to replace the code with `while true do end` to cause an infinite loop to prevent any further code from running, albeit that could lead to crashes. And have the print in front of the while loop.